protostar/testing/fuzzing/strategies/uint256.py

```diff
@@ -20,7 +20,9 @@ def _get_high(uint256: int):
 
 class Uint256StrategyDescriptor(StrategyDescriptor):
     def build_strategy(self, cairo_type: CairoType) -> SearchStrategy[int]:
-        if not isinstance(cairo_type, TypeStruct):  # TODO: FIND BETTER CHECK
+        if not isinstance(cairo_type, TypeStruct) and str(cairo_type.scope).endswith(
```

Comment:
I would test fully qualified path, so that user-defined structs named the same won't be picked up by this

Reply:
After manual testing full path seems to be pretty inconsistent, I think the current approach is good enough.

Reply:
What's inconsistent here? Do you observe different paths to the type? 🤨

Reply:
It turns out that `starknet_py.cairo.felt.is_uint256` is using CairoTypes from ABI which has a different naming scheme.